### PR TITLE
Fix Documentation on MetricConfig field: tagsToRemove

### PR DESCRIPTION
**Pull request by crhuber**
Fixes Documentation on MetricConfig field: tagsToRemove.
Currently if you configure your manifest file with tagsToRemove, it has no effect on the tags that are removed by Istio-proxy ie:

has no effect on tags:
```
    telemetry:
      enabled: true
      v1:
        enabled: false
      v2:
        enabled: true
        prometheus:
          configOverride:
            inboundSidecar:
              debug: false
              stat_prefix: istio
              metrics:
                - name: requests_total
                  tagsToRemove: ["destination_principal", "source_principal"]
```

However, this works:

```
    telemetry:
      enabled: true
      v1:
        enabled: false
      v2:
        enabled: true
        prometheus:
          configOverride:
            inboundSidecar:
              debug: false
              stat_prefix: istio
              metrics:
                - name: requests_total
                  tags_to_remove: ["destination_principal", "source_principal"]
```

This PR updates the documentation to reflect that `tags_to_remove` is the correct field name

[ ] Configuration Infrastructure
[x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by crhuber** [comment]
@googlebot I signed it!

File: content/en/docs/reference/config/proxy_extensions/stats/index.html [hunk 1]
```diff
@@ -54,7 +54,7 @@ <h2 id="MetricConfig">MetricConfig</h2>
 </td>
 </tr>
 <tr id="MetricConfig-tags_to_remove">
-<td><code>tagsToRemove</code></td>
+<td><code>tags_to_remove</code></td>
```
Comment (ericvn) [inline content/en/docs/reference/config/proxy_extensions/stats/index.html:57]: This change comes from a file in another repo. Making the change here will be overwritten the next day.

I think the update might need to be here: https://github.com/istio/proxy/blob/master/extensions/stats/config.proto#L44

**Review by ericvn — CHANGES_REQUESTED** [review]

File: content/en/docs/reference/config/proxy_extensions/stats/index.html (hunk 1 above)
Comment (ericvn) [reply content/en/docs/reference/config/proxy_extensions/stats/index.html:57]: I believe that comes from here: https://github.com/istio/proxy/blob/master/extensions/stats/config.proto#L44

I don'r know enough about the generation from the .proto to the .pb.html.

**Comment by bianpengyuan** [comment]
https://github.com/istio/proxy/pull/3191 should supersede this.

**Comment by ericvn** [comment]
https://github.com/istio/proxy/pull/3191 does have a change for the master branch which will update preliminary.istio.io. I did ask about cherrypicking it back to 1.9 which will fix istio.io (at least latest).

**Comment by ericvn** [comment]
Closing this PR as istio/proxy#3191 should fix the problem and it is incorrect to fix in this repo.

**Pull request closed without merging**